From bb2d29750f757dbff4ef371a957d9316e24321c4 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Mon, 17 Apr 2017 11:11:59 -0700
Subject: [PATCH] DOCKER-1036
 node-docker-registry-client/examples/v2/getManifest.js should dump the raw
 manifest response string

---
 examples/v2/getManifest.js | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/examples/v2/getManifest.js b/examples/v2/getManifest.js
index 784803f..20180d8 100644
--- a/examples/v2/getManifest.js
+++ b/examples/v2/getManifest.js
@@ -37,7 +37,8 @@ mainline({cmd: cmd}, function (log, parser, opts, args) {
         maxSchemaVersion: (opts.schema || 1)
     });
     var tagOrDigest = rar.tag || rar.digest;
-    client.getManifest({ref: tagOrDigest}, function (err, manifest, res) {
+    client.getManifest({ref: tagOrDigest}, function (err, manifest, res,
+                                                     manifestStr) {
         client.close();
         if (err) {
             mainline.fail(cmd, err, opts);
@@ -45,6 +46,6 @@ mainline({cmd: cmd}, function (log, parser, opts, args) {
         console.error('# response headers');
         console.error(JSON.stringify(res.headers, null, 4));
         console.error('# manifest');
-        console.log(JSON.stringify(manifest, null, 4));
+        console.log(manifestStr);
     });
 });
-- 
2.21.0

